fastlane/lanes/codesign.rb

```diff
@@ -87,6 +87,18 @@
     )
   end
 
+  # Downloads all the required certificates and profiles (using `match``) for the Reader App Store builds.
+  # Optionally, it can create any new necessary certificate or profile.
+  #
+  # @option [Boolean] readonly (default: true) Whether to only fetch existing certificates and profiles, without generating new ones.
+  #
+  lane :update_certs_and_profiles_app_store_reader do |readonly: true|
+    update_code_signing_app_store(
+      app_identifiers: ALL_READER_BUNDLE_IDENTIFIERS,
+      readonly: readonly
+    )
+  end
```

Comment:
I named it with `..._app_store_reader` instead of `..._reader_app_store`
like the other lanes for Jetpack and WordPress because I find that
suffixing the app name leads to better code completion and homogeneous
API.

```
..._app_store_jetpack
..._app_store_reader
..._app_store_wordpress
```

This is not the time to change the other lane names, however.